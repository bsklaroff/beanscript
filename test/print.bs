print(23)
print(-47)
print(True)
print(False)
