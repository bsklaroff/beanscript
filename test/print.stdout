23
-47
True
False
